### PR TITLE
Reorganize consistency check sections

### LINKED ISSUE #26: Unique DoH path attack
Some options include: only allow common DoH paths (such as the de-facto default "/dns-query{?dns}");
performing consistency checks by fetching the information about the resolver over multiple resolution paths; or coordinating with a
trusted oblivious relay to validate that DoH paths are common across clients using the same gateway.

Comment> The consistency check discussed above in security considerations section is not clear to me. For example, in case of DNR, the client will use DHCP/RA to learn the DoH path and it can use discovered ADN or Do53 to do the the SVCB lookup to cross-check both DoH paths are matching. 

I don't get how the client will co-ordinate with a oblivious relay to validate that DoH path. Is the client supposed to use the de-facto URI path to retrieve the actual URI path from the Oblivious DoH service.

### PR DESCRIPTION
**Pull request by tfpauly**
Closes #26 

Basing text on the recommendation in the IETF 116 meeting of OHAI

### CODE REVIEW

File: draft-ietf-ohai-svcb-config.md [hunk 1]
```diff
@@ -300,20 +300,25 @@ always checking for oblivious gateway configuration {{config-fetch}}
 on the well-known gateway location {{gateway-location}}.
 Use of encrypted DNS along with DNSSEC can also be used as a mitigation.
 
+When clients fetch a gateway's configuration ({{config-fetch}}),
+they can expose their identity in the form of an IP address if they do not
+connect via a proxy or some other IP-hiding mechanism. In some circumstances,
+this might not be a privacy concern, since revealing that a particular
+client IP address is preparing to use an Oblivious HTTP service can be
+expected. However, if a client is otherwise trying to obfuscate its IP
```
Comment (chris-wood) [inline draft-ietf-ohai-svcb-config.md]:
```suggestion
expected. However, if a client is otherwise trying to hide its IP
```

Comment (chris-wood) [reply draft-ietf-ohai-svcb-config.md]: ... for consistency with "IP-hiding" above.

File: draft-ietf-ohai-svcb-config.md [hunk 2]
```diff
@@ -300,20 +300,25 @@ always checking for oblivious gateway configuration {{config-fetch}}
...
+address or location (and not merely decouple its specific requests from its
+IP address), or revealing its IP address will increase the risk of a key
+targeting attack (if a gateway service is trying to differentiate traffic
+across client IP addresses), a proxy or similar mechanism can be used to fetch
```
Comment (chris-wood) [inline draft-ietf-ohai-svcb-config.md]:
```suggestion
IP address), or if revealing its IP address facilitates key targeting attacks 
(if a gateway service uses IP addresses to associate specific configurations
with specific clients), a proxy or similar mechanism can be used to fetch
```

File: draft-ietf-ohai-svcb-config.md [hunk 3]
```diff
@@ -325,20 +330,21 @@ There are several approaches clients can use to mitigate key targeting
...
-address or location (and not merely decouple its specific requests from its
-IP address), or revealing its IP address will increase the risk of a key
-targeting attack (if a gateway service is trying to differentiate traffic
-across client IP addresses), a proxy or similar mechanism can be used to fetch
-the gateway's configuration.
+## dohpath Targeting Attacks
+
+For oblivious DoH servers, an attacker could use unique `dohpath` values
+to target or identify specific clients. This attack is very similar to
+the key targeting attack described above.
```
Comment (chris-wood) [inline draft-ietf-ohai-svcb-config.md]:
```suggestion
the generic OHTTP key targeting attack described above.
```

Comment (chris-wood) [reply draft-ietf-ohai-svcb-config.md]: ... to make it more clear why there's a separate section for the dohpath.

File: draft-ietf-ohai-svcb-config.md [hunk 4]
```diff
@@ -325,20 +330,21 @@ There are several approaches clients can use to mitigate key targeting
...
+
+Clients SHOULD mitigate such potential attacks. This can be done with
```
Comment (chris-wood) [inline draft-ietf-ohai-svcb-config.md]:
```suggestion
Clients SHOULD mitigate such attacks. This can be done with a
```

**Review by chris-wood — APPROVED** [review]

**Review by tireddy2 — APPROVED** [review]
Update  looks good.

File: draft-ietf-ohai-svcb-config.md [hunk 5]
```diff
@@ -300,20 +300,25 @@ always checking for oblivious gateway configuration {{config-fetch}}
...
 Use of encrypted DNS along with DNSSEC can also be used as a mitigation.
 
+When clients fetch a gateway's configuration ({{config-fetch}}),
+they can expose their identity in the form of an IP address if they do not
+connect via a proxy or some other IP-hiding mechanism. In some circumstances,
+this might not be a privacy concern, since revealing that a particular
+client IP address is preparing to use an Oblivious HTTP service can be
+expected. However, if a client is otherwise trying to hide its IP
+address or location (and not merely decouple its specific requests from its
+IP address), or if revealing its IP address facilitates key targeting attacks
```
Comment (tfpauly) [inline draft-ietf-ohai-svcb-config.md]:
```suggestion
IP address), or if revealing its IP address facilitates key targeting attacks
```

**Pull request merged**